test/test_broker.hpp

```diff
@@ -839,6 +846,37 @@ class test_broker {
 
         auto& ep = *spep;
 
+        std::vector<bool> new_sub;
+        new_sub.reserve(entries.size());
+
+        auto add_or_overwrite_sub =
+            [&](MQTT_NS::buffer topic, MQTT_NS::qos qos_value, MQTT_NS::rap rap_value = MQTT_NS::rap::dont) {
+                auto& idx = subs_online_.get<tag_con_topic>();
+                auto r = idx.equal_range(std::make_tuple(spep, topic));
```

Comment:
Instead of equal_range, use idx.find(std::make_tuple(spep, topic)).

Reply:
No, it is for emplace_hint.